src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue27998.cs

```diff
@@ -0,0 +1,39 @@
+﻿//#if WINDOWS
+using NUnit.Framework;
+using OpenQA.Selenium.Internal;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue27998 : _IssuesUITest
+	{
+		private const string _itemsId = "ItemsId";
+		private const string _lastItemId = "LastItemId";
+		private const string _firstItemId = "FirstItemId";
+		private const string _lastItemText = "INSIDE SCROLL 27";
+
+		public Issue27998(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "[Windows] ScrollView is not scrolling to the bottom if in grid with *,auto Width";
+
+		[Test]
+		[Category(UITestCategories.Cells)]
+		[Category(UITestCategories.ScrollView)]
+		[Category(UITestCategories.Border)]
+		public void GridAutosStarsScollToEndDisplaysLastItem()
```

Comment:
The test is failing on Android, iOS and Mac.

```
Assert.That(lastItem.IsDisplayed().Equals(false), Is.True)
Expected: True
But was:  False
```

Could you take a look?

Reply:
Sorry for not responding for a long while.
I have been drowning in uni activities.
But was also stuck as the compilation failed for ios when paired with a macincloud device.
I will open a discussion regarding the issue I faced.

But my testcase succeds now for android.

Open tasks for myself:
- check if testcase succeeds on ios and mac
- investigate the other failing tests